meta-genivi-dev/meta-genivi-dev/recipes-sota/genivi-swm/files/0001-common-settings-update-DB_URL-to-var-run-swm-swlm.sq.patch

```diff
@@ -0,0 +1,41 @@
+From 69ee3d50bd26ff6708327558d0e7933136631c6c Mon Sep 17 00:00:00 2001
```

Comment:
Regarding swm we ought to change the original project source and drop the patch.  But since this is going to be reworked anyway, maybe there's no point in optimizing now.

Reply:
My idea was:

- Merge here
- Create pull-request upstream
- Drop patch here once upstream was updated

Mostly because I was unsure who has control (maintainership) of the upstream repo and avoiding "blocks".

Reply:
Yeah, we are thinking the same here - I just noted it for the future.